.github/workflows/ci.yaml

```diff
@@ -33,6 +33,21 @@ jobs:
           components: clippy
       - uses: actions-rs/cargo@v1
         with:
+          command: clippy
+          args: -- -D warnings
```

Comment:
It's a shame there's no "except features" here so that a new feature would need to opt out from being stable tested. But then again let's be honest how many new features will this lib ever get 😁 The GH matrix might make it more visible though.